orchestrator-salt/src/main/resources/salt/salt/cloudera/csd/csd-downloader.sh

```diff
@@ -3,11 +3,14 @@
 set -ex
 
 mkdir -p /opt/cloudera/csd
+rm -f /opt/cloudera/csd/*
 cd /opt/cloudera/csd
 
 {% if salt['pillar.get']('cloudera-manager:csd-urls') %}
 {% for url in salt['pillar.get']('cloudera-manager:csd-urls') %}
+{% if salt['pillar.get']('cloudera-manager:csd-urls') %}
```

Comment:
What is this extra if condition for? There should be a test -f here to avoid downloading the csd with every highstate.

Reply:
Sorry, this was for testing